documentation/smg/14_documentation.md

```diff
@@ -0,0 +1,80 @@
+# Documentation
+
+This document will outline the process surrounding the documentation currently housed in the `horace-docs` repository. 
+It assumes a basic knowledge of reStructured text (reST) format (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html)
```

Comment:
From the website it is consistently referred to as "reStructuredText" rather than "reStructured text"